step.yml

```diff
@@ -52,6 +52,12 @@ inputs:
       summary: ""
       description: "Project ARNs can be obtained using the AWS CLI 'devicefarm list-projects' command."
       is_required: true
+  - billingMethod: ""
```

Comment:
I think you can change this to `billingMethod: "METERED"` to set the default to metered